### PR TITLE
Bugs LVJAVJ-62,78,80 are fixed

### PR DESCRIPTION
**Pull request by aprudyus**
Error messages in registration form are displayed in red color from now on.

Bug with birthdate is also fixed. User can't set birthdate to some values like 22/05/2333.

Some code design fixes.

### CODE REVIEW

File: web/src/main/resources/public/src/app/registration/registration_user/googleplace.directive.ts [hunk 1]
```diff
@@ -1,38 +1,38 @@
...
+    invokeEvent(place: Object) {
+        this.setAddress.emit(place);
+    }
 
 
-  onInputChange() {
-  }
+    onInputChange() {
+    }
 }
```
Comment (KostyantynPanchenko) [inline web/src/main/resources/public/src/app/registration/registration_user/googleplace.directive.ts]: add new line at the end of file

File: web/src/main/resources/public/src/app/registration/registration_user/register.service.ts [hunk 2]
```diff
@@ -66,22 +66,4 @@ export class RegisterService {
...
-//     let headers=new Headers({'Content-Type':'application/json'});
-// return this.http.put(this._pathUrl2,JSON.stringify(user),{headers:headers});
-//}
-
-// sendU(user: User){
-//     console.log()
-//     return this.userId;
-// }
\ No newline at end of file
+}
```
Comment (KostyantynPanchenko) [inline web/src/main/resources/public/src/app/registration/registration_user/register.service.ts]: add new line at hte end of file

File: web/src/main/resources/public/src/shared/models/user_registration.ts [hunk 3]
```diff
@@ -14,16 +14,6 @@ export class UserRegistration {
...
-        // this.birthDate = new String();
-        // this.email = new String();
-        // this.phoneNumber = new String();
-        // this.gender = new String();
-        // this.activated = new Boolean();
-        // this.apartmentId = new Number();
-        // this.osbbId= new Number();
         this.role = 1;
     }
 }
```
Comment (KostyantynPanchenko) [inline web/src/main/resources/public/src/shared/models/user_registration.ts]: add new line at the end of file

**Review by KostyantynPanchenko — COMMENTED** [review]
Please add description of your tasks - so we will know what you should have fixed

**Review by KostyantynPanchenko — COMMENTED** [review]
Please decribe your tasks

**Review by sergvasylchak — APPROVED** [review]

**Review by KostyantynPanchenko — APPROVED** [review]
+

**Pull request merged**